controllers/workspace/status.go

```diff
@@ -158,7 +158,7 @@ func checkServerStatus(workspace *dw.DevWorkspace) (ok bool, err error) {
 func getMainUrl(exposedEndpoints map[string]v1alpha1.ExposedEndpointList) string {
 	for _, endpoints := range exposedEndpoints {
 		for _, endpoint := range endpoints {
-			if endpoint.Attributes.GetString(string(v1alpha1.TypeEndpointAttribute), nil) == "ide" {
+			if endpoint.Attributes.GetString(string(v1alpha1.TypeEndpointAttribute), nil) == "main" {
```

Comment:
Consider putting this value in `constants.go`

Reply:
What about a dedicated type for EndpointType?
I did this way https://github.com/devfile/devworkspace-operator/pull/430/commits/e9dce8d8494e8e91fd2bb992c6dae3e122b7b9a8 but if you vote just for string in constants.go - let me know and I'll rework since I don't have a strong opinion here.